UserService/src/main/java/com/beaconfire/userservice/config/SecurityConfig.java

```diff
@@ -30,7 +30,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .csrf().disable() // Disable CSRF protection
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and() // Use stateless sessions
                 .authorizeRequests()
-                .antMatchers("/user-service/user/authenticate", "/user-service/user/create").permitAll()
+                .antMatchers("/user-service/user/authenticate", "/user-service/user/create", "/user-service/user/verify").permitAll()
```

Comment:
Are we allowing anyone to access this endpoint without JWT token?

Reply:
Only SuperAdmin can access this endpoint, I will add @PreAuthorize("hasAnyAuthority('ROLE_SADMIN')") for that end point.

Reply:
Got it.